### PR TITLE
[office-js] Excel 1.8 Updates

### PR DESCRIPTION
**Pull request by AlexJerabek**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://appsforoffice.microsoft.com/lib/beta/hosted/office.d.ts
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by AlexJerabek** [comment]
**Maintainers**: please do not merge until this is reviewed by @Zlatkovsky. Thanks!

**Review by Rick-Kirkham — CHANGES_REQUESTED** [review]

File: types/office-js/index.d.ts
Comment (AlexJerabek) [reply types/office-js/index.d.ts]: These are LoadOptions, so the comments are based off the original properties, not Boolean selectors deciding loading.

Comment (Rick-Kirkham) [reply types/office-js/index.d.ts]: OK. That explains this item. We have an item to fix that. Because of the missing lines that I emailed you about, I couldn't see what the class was.

File: types/office-js/index.d.ts
Comment (Rick-Kirkham) [reply types/office-js/index.d.ts]: @AlexJerabek If this is a *LoadOptions class, that explains why it is Boolean, but in this case there's a meta mismatch between the description and the property name. Description says that it is a format object but the property name is "id".

**Review by Rick-Kirkham — APPROVED** [review]

File: types/office-js/index.d.ts
Comment (Zlatkovsky) [inline types/office-js/index.d.ts]: Are all these changes to 'OneNote.run`/`Visio.run` by design?

**Review by Zlatkovsky — COMMENTED** [review]
Looks good to me (though please double-check re. my comment).
Also, we need to wait on one thing, to see if one of the APIs needs to be postponed.  Alex, I'll add you to the email thread.

File: types/office-js/index.d.ts
Comment (AlexJerabek) [reply types/office-js/index.d.ts]: Yeah, they looked to be documenting the wrong method. We also wanted the most useful signature at the end.

**Pull request merged**
